js-pkg/apps/demos/src/components/webrtc.js

```diff
@@ -137,7 +137,11 @@ export const useWebRTCConnection = (p1, p2, connSetupArray) => {
   }, [signalingMessages])
 
   const getLatency = async () => {
-    const stats = await connRef.current?.getStats(null)
+    try {
+      //NOTE: this will sometimes be undefined since this code may be run
+      // before a connection is established. that's OK.
+      var stats = await connRef.current?.getStats(null)
```

Comment:
I'm confused by the comment - `getStats()` returns `undefined` before a connection is established? Why is the `try/catch` necessary?

Reply:
the optional accessor throws an error

Reply:
but I guess this is unwieldy, better to just check that's defined.